src/Services/ServicesFactory.php

```diff
@@ -2,28 +2,47 @@
 
 namespace SMW\Services;
 
-use Closure;
 use Onoi\CallbackContainer\CallbackContainerFactory;
 use Onoi\CallbackContainer\ContainerBuilder;
-use Parser;
+use Onoi\EventDispatcher\EventDispatcher;
 use ParserOutput;
+use Psr\Log\LoggerInterface;
+use SMW\CacheFactory;
+use SMW\Connection\ConnectionManager;
+use SMW\ContentParser;
+use SMW\DataItemFactory;
+use SMW\DataUpdater;
+use SMW\DataValueFactory;
+use SMW\DeferredCallableUpdate;
+use SMW\DeferredTransactionalCallableUpdate;
+use SMW\EntityCache;
+use SMW\EventHandler;
```

Comment:
This sort of change here costs me a lot of grievance (in the past and as of now) as it forces unnecessary work on rebasing PRs.

Reply:
Merge conflicts are the worst

Maybe I should have disabled the "reorder alphabetically" feature before running the cleanup command for this particular file, or for SMW in general. I'm used to working in projects where this is cleanup a non-event and rarely or never causes merge conflicts.